app/models/UserSchema/UserSchema.js

```diff
@@ -0,0 +1,47 @@
+import mongoose from "mongoose";
+
+const UserSchema = new Schema({
+  email: {
+    type: String,
+    required: [true, "Email is required"],
+    lowercase: true,
+    trim: true,
+    unique: true,
+  },
+  username: {
+    type: String,
+    required: true,
+    minLength: [3, "At least 3 characters"],
+  },
+  city: {
+    type: String,
+    required: true,
+    minLength: [3, "At least 3 characters"],
+  },
+  phone: {
+    type: Number,
+    required: true,
+    minLength: [8, "At least 8 characters"],
+    maxlength: [10, "Maximum 10 characters"],
+  },
+  announcements: [
+    { type: mongoose.Types.ObjectId, required: true, ref: "Announcement" },
```

Comment:
When I log in to the app first time, I don't have any announcements. So, is it required?